recipe/meta.yaml

```diff
@@ -1,20 +1,22 @@
 {% set pip_name = 'avro' %}  # [py27]
 {% set pip_name = 'avro-python3' %}  # [py3k]
-{% set version = "1.9.2" %}
+{% set version2 = "1.9.2" %}  # {py27}
+{% set version3 = "1.9.2.1" %}  # {py3k}
```

Comment:
This is required because the source package created an extra `.1` update only for the py3 package.